creator-node/src/pinCID.js

```diff
@@ -0,0 +1,45 @@
+const { CID } = require('ipfs-http-client-latest')
+
+const config = require('./config')
+const { logger } = require('./logging')
+
+/**
+ * Pin CIDs from config
+ * @param {*} ipfs 
+ */
+const pinCID = async (ipfs) => {
+  try {
+    const addCIDs = config.get('pinAddCIDs')
+      .split(',')
+      .filter(cid => cid !== '')
+      .map(cid => new CID(cid))
+
+    for (const cid of addCIDs) {
+      try {
+        await ipfs.pin.add(cid, { recursive: true })
```

Comment:
what is the behavior if this operation never completes? jw if there are any adverse consequences if we try to pin an unavailable CID and this hangs indefinitely

Reply:
Optional timeout: https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/PIN.md#options

Maybe we could add some super large timeout

Reply:
There's also ipfs.pin.addAll

Reply:
The pinAll has an input as an async iterator. I think it'll be clearer/more readable w/ a for loop. 
Testing on staging. If we peer, it should pin add in ~4 seconds.